persistence.js

```diff
@@ -33,6 +31,8 @@ function RedisPersistence (opts) {
   this.maxSessionDelivery = opts.maxSessionDelivery || 1000
   this._db = new Redis(opts)
 
+  this.messageIdCache = {}
```

Comment:
Considering there are many Aedes instances, and the connections are non-sticky. The cache size will keep increasing.